Cargo.lock

```diff
@@ -7768,9 +7768,9 @@ dependencies = [
 
 [[package]]
 name = "url"
-version = "2.5.3"
+version = "2.5.4"
```

Comment:
This is the source of the unexpected test failures.

Reply:
Oh, interesting. I've undone the upgrade and added an exception for `url@2.5.3` which has apparently been yanked.